Makefile

```diff
@@ -352,26 +369,26 @@ build-tests:
 
 unit-test: clean-test-results
 	@printf $(COLOR) "Run unit tests..."
-	@go test $(UNIT_TEST_DIRS) -shuffle on -timeout=$(TEST_TIMEOUT) $(TEST_TAG_FLAG) $(TEST_ARGS) 2>&1 | tee -a test.log
+	@go test $(UNIT_TEST_DIRS) $(COMPILED_TEST_ARGS) 2>&1 | tee -a test.log
```

Comment:
personally I'd rather see the full command here and everywhere, so it's easy to copy-paste with an extra `-run` or whatever to narrow down to one test, but be confident I'm using the right flags

Reply:
Same, I'll raise this with the team and confirm there's no objections to change.